### PR TITLE
Feature/quiet submission

### LINKED ISSUE #2673: Add "quiet" way to ingest reports/create incidents without them going to front page
Sometimes we have reports that substantiate incidents and add to meaningful knowledge about said incidents, but may have sensationalist titles that we don't want to promote as-is on the homepage. (#2563 addresses this from a design perspective)
Similarly, we may wish to index historical incidents without taking the emphasis off novel or more current ones or accidentally implying that they were more recent events.

To solve this, we should add a control for editors to mark the ingestion of reports or creation of an incident as "quiet" so that the landing page and other rotating content don't promote these objects. This would also allow us to preserve date metadata (submission date, creation date, date of incident).

### PR DESCRIPTION
**Pull request by clari182**
- closes #2673 

Test preview link: https://pr-46--clarayoudale-staging.netlify.app/

Latest action: https://github.com/clari182/aiid/actions/runs/8156060421

New quiet option in submission form and report edit form:
image

If quiet is selected, the report will not appear in the home page carousel of latest reports

### CODE REVIEW

**Comment by clari182** [comment]
@kepae @datherton09 the quiet option is ready for a test round. Let me know if you think it looks ok the way it does.
image

That option is on the submission form and report edit form

The test url is: https://pr-46--clarayoudale-staging.netlify.app/
Ping me for login credentials

File: site/gatsby-site/cypress/e2e/unit/functions/promoteSubmissionToReport.cy.js
Comment (kepae) [inline site/gatsby-site/cypress/e2e/unit/functions/promoteSubmissionToReport.cy.js:1]: is there a meaningful way to add a test case where `quiet: true`, and check that the expected report has the field propagated?

**Review by kepae — CHANGES_REQUESTED** [review]
LGTM, just curious about how to best test this.

File: site/gatsby-site/cypress/e2e/unit/functions/promoteSubmissionToReport.cy.js
Comment (clari182) [reply site/gatsby-site/cypress/e2e/unit/functions/promoteSubmissionToReport.cy.js:1]: Updated a test to include the quiet field

**Review by kepae — APPROVED** [review]
Thanks for adding the test!

**Comment by kepae** [comment]
@clari182 thanks for updating/syncing. :-) Good to merge to staging?

**Comment by clari182** [comment]
@kepae Yes we're good to merge!

**Pull request merged**